### PR TITLE
AdGuard Base Filter

### PR DESCRIPTION
**Pull request by O7zSO31IVg**

### CODE REVIEW

**Comment by ignoramous** [comment]
Thanks. There already another [AdGuard filter](https://github.com/serverless-dns/blocklists/blob/c858b3a09222ef6297db2df4e9fe8921d77e5e66/blocklistConfig.json#L827), currently. Is this 'base filter' list a subset? Also, why/when would one use it over the other?

File: blocklistConfig.json [hunk 1]
```diff
@@ -1496,6 +1496,14 @@
       "url": "https://raw.githubusercontent.com/Clefspeare13/pornhosts/master/download_here/strict/0.0.0.0/hosts",
       "pack": ["try again"]
     },    
+    {
+      "vname": "AdGuard Base Filter",
+      "format":"adguard",
```
Comment (ignoramous) [inline blocklistConfig.json:1501]: format is one of `domains`, `hosts`, or `abp` (adblock plus)

**Comment by O7zSO31IVg** [comment]
More blocking options

File: blocklistConfig.json (hunk 1 above)
Comment (O7zSO31IVg) [reply blocklistConfig.json:1501]: adguard syntax

**Comment by ignoramous** [comment]
Gotcha. Btw, I don't think RethinkDNS can parse the 'adguard syntax'. Taggin @santhosh-ponnusamy for his inputs.

**Comment by O7zSO31IVg** [comment]
> Gotcha. Btw, I don't think RethinkDNS can parse the 'adguard syntax'. Taggin @santhosh-ponnusamy for his inputs.

what formats can RethinkDNS parse?

**Comment by santhosh-ponnusamy** [comment]
> > Gotcha. Btw, I don't think RethinkDNS can parse the 'adguard syntax'. Taggin @santhosh-ponnusamy for his inputs.
> 
> what formats can RethinkDNS parse?

currently, supported file parsing is host, domain, and ABP(AdBlock Plus).
from ABP files, we extract only domain names.

**Pull request closed without merging**